### PR TITLE
Python: `ShortRF` Properties

### PR DESCRIPTION
**Pull request by ax3l**
Add element properties for `ShortRF`, so it can be modified after variable creation, too.

### CODE REVIEW

**Comment by ax3l** [comment]
@EZoni volunteered to complete the other leftover elements in a separate PR :pray:

**Review by cemitch99 — APPROVED** [review]
Looks great, thanks.

**Review by EZoni — APPROVED** [review]

**Review by cemitch99 — APPROVED** [review]
Looks great.

File: src/python/elements.cpp [hunk 1]
```diff
@@ -969,6 +969,21 @@ void init_elements(py::module& m)
...
+            "Normalized RF voltage V = maximum energy gain/(m*c^2)"
+        )
+        .def_property("freq",
+            [](ShortRF & short_rf) { return short_rf.m_freq; },
+            [](ShortRF & short_rf, int freq) { short_rf.m_freq = freq; },
+            "RF frequency in Hz"
+        )
+        .def_property("phase",
+            [](ShortRF & short_rf) { return short_rf.m_phase; },
+            [](ShortRF & short_rf, int phase) { short_rf.m_phase = phase; },
```
Comment (EZoni) [inline src/python/elements.cpp:984]: Actually I just noticed this, should `freq` and `phase` be `amrex::ParticleReal` too, as opposed to `int`? If so, I can fix it in the follow-up PR.

Comment (cemitch99) [reply src/python/elements.cpp:984]: Thanks for catching this!  Those should definitely be amrex::ParticleReal.

Comment (ax3l) [reply src/python/elements.cpp:984]: Ooops, yes please. Copy pasted this, dang!

Comment (EZoni) [reply src/python/elements.cpp:984]: @ax3l 
I will fix it in the upcoming PR. Question, which I guess is pybind11-related: why does the compiler not complain about/catch this discrepancy between types?

Comment (ax3l) [reply src/python/elements.cpp:984]: I am surprised as well, should issue a narrowing warning.
https://godbolt.org/z/f6bE3xPPE

Thx for the fix in #647

**Pull request merged**